proxies/java/src/main/java/com/dvc_harness/proxy/controller/ProxyController.java

```diff
@@ -69,7 +71,96 @@ public BaseResponse client(@RequestBody ClientRequestBody body, HttpServletRespo
             return new MessageResponse("success");
         } catch (Exception e) {
             response.setStatus(200);
-            return new ExceptionResponse(e.getMessage());
+            return new ExceptionResponse(e);
+        }
+    }
+
+    @PostMapping("/*/*")
+    public BaseResponse location(
+        HttpServletRequest request,
+        @RequestBody LocationRequestBody body,
+        HttpServletResponse response
+    ) {
+        response.addHeader("CloudClients", DataStore.CloudClients.toString());
```

Comment:
oops, that doesn't need to be there

Reply:
everything makes sense to me!:) It would be safer for someone else who is familiar with the API design to review it as well, but I'll still send my tick out for now :)